iam-login-service/src/test/java/it/infn/mw/iam/test/oauth/scope/StructuredScopeTestSupportConstants.java

```diff
@@ -40,7 +43,14 @@ interface StructuredScopeTestSupportConstants {
   static final String DEVICE_USER_VERIFY_URL = "http://localhost:8080/device/verify";
   static final String DEVICE_USER_APPROVE_URL = "http://localhost:8080/device/approve";
 
+  static final String WRONG_VERIFICATION_URI_COMPLETE = "/device?user_code=";
```

Comment:
This is not a URI (and don't know what "complete") means.
I would keep the name consistent with the other `[SOMETHING]_ENDPOINT` variables (even though, technically speaking, they are not even endpoints 😅)

Reply:
I replaced this constant with DEVICE_CODE_USER_ENDPOINT + "?user_code" cause it was used only by one test